airbyte-integrations/connectors/source-shopify/acceptance-test-config.yml

```diff
@@ -1,39 +1,52 @@
 connector_image: airbyte/source-shopify:dev
-tests:
+test_strictness_level: high
+acceptance_tests:
   spec:
-    - spec_path: "source_shopify/spec.json"
+    tests:
+      - spec_path: "source_shopify/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
-    - config_path: "secrets/config_old.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config_old.json"
-      status: "failed"
-    - config_path: "secrets/config_oauth.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_oauth_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
+      - config_path: "secrets/config_old.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config_old.json"
+        status: "failed"
+      - config_path: "secrets/config_oauth.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_oauth_config.json"
+        status: "failed"
   discovery:
-    - config_path: "secrets/config.json"
-    - config_path: "secrets/config_old.json"
-    - config_path: "secrets/config_oauth.json"
+    tests:
+      - config_path: "secrets/config.json"
+      - config_path: "secrets/config_old.json"
+      - config_path: "secrets/config_oauth.json"
   basic_read:
-    - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/configured_catalog.json"
-      timeout_seconds: 7200
-      # some streams hold data only for some time, therefore certain streams could be empty while sync.
-      # 'abandoned_checkouts' stream holds data up to 1 month.
-      empty_streams: ["abandoned_checkouts", "balance_transactions"]
+    tests:
+      - config_path: "secrets/config.json"
+        timeout_seconds: 7200
+        expect_records:
+          path: "integration_tests/expected_records.txt"
+        empty_streams:
+          - name: "shop"
+            bypass_reason: "The stream contains sensitive personal information. Skipped."
```

Comment:
Could you please let me know in DM on Slack what's exactly sensitive? Can we change these information on our  Shopify account to make these less sensitive?

Reply:
It's the info about "Who" created the `Shop` and his personal info like: Phone, Email, etc. 
That info should be real I  guess to confirm with the owner if something is happening.